Missing @rollup/rollup-win32-x64-msvc

### Overview of the problem

**Buefy** version: [0.2.1]
**Vuejs** version: [3.4.13]
**OS/Browser**: Windows/na

### Description

`rollup` needs a binary specifically built for the host OS.

### Steps to reproduce

1. `npm ci` on Windows
2. `npm -w docs run dev`

### Expected behavior

No error.

### Actual behavior

The following error happened.

```
Error: Cannot find module @rollup/rollup-win32-x64-msvc. npm has a bug related to optional dependencies (https://github.com/npm/cli/issues/4828). Please try npm i again after removing both package-lock.json and node_modules directory.
    at requireWithFriendlyError (E:\GitHub\MyOrganizations\ntohq\buefy-next\node_modules\rollup\dist\native.js:64:9)
    at Object.<anonymous> (E:\GitHub\MyOrganizations\ntohq\buefy-next\node_modules\rollup\dist\native.js:73:76)
    at Module._compile (node:internal/modules/cjs/loader:1562:14)
    at Object..js (node:internal/modules/cjs/loader:1699:10)
    at Module.load (node:internal/modules/cjs/loader:1313:32)
    at Function._load (node:internal/modules/cjs/loader:1123:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
    at cjsLoader (node:internal/modules/esm/translators:263:5)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:196:7) {
  [cause]: Error: Cannot find module '@rollup/rollup-win32-x64-msvc'
  Require stack:
```

--- a/package.json
+++ b/package.json
@@ -30,7 +30,8 @@
     "vue-tsc": "^2.2.2"
   },
   "optionalDependencies": {
-    "@rollup/rollup-linux-x64-gnu": "^4.34.8"
+    "@rollup/rollup-linux-x64-gnu": "^4.34.8",
+    "@rollup/rollup-win32-x64-msvc": "^4.40.0"
   },
   "repository": {
     "type": "git",

--- a/packages/buefy-next/rollup.config.mjs
+++ b/packages/buefy-next/rollup.config.mjs
@@ -8,7 +8,7 @@ import vue from '@vitejs/plugin-vue'
 import fs from 'node:fs'
 import path from 'node:path'
 
-import pack from '../../package.json' assert { type: 'json' }
+import pack from '../../package.json' with { type: "json" }
 
 const babelConfig = {
     exclude: 'node_modules/**',

--- a/packages/buefy-next/rollup.config.mjs
+++ b/packages/buefy-next/rollup.config.mjs
@@ -8,7 +8,7 @@ import vue from '@vitejs/plugin-vue'
 import fs from 'node:fs'
 import path from 'node:path'
 
-import pack from '../../package.json' with { type: "json" }
+import pack from '../../package.json' with { type: 'json' }
 
 const babelConfig = {
     exclude: 'node_modules/**',

